setup.py

```diff
@@ -1,118 +1,144 @@
-#!/usr/bin/env python
-"""Ibis setup module."""
-import pathlib
-import sys
+# -*- coding: utf-8 -*-
+from setuptools import setup
 
-from setuptools import find_packages, setup
-
-import versioneer
-
-LONG_DESCRIPTION = """
-Ibis is a productivity-centric Python big data framework.
-
-See http://ibis-project.org
-"""
-
-VERSION = sys.version_info.major, sys.version_info.minor
-
-impala_requires = ['hdfs>=2.0.16', 'sqlalchemy>=1.3', 'requests']
-impala_requires.append('impyla[kerberos]>=0.15.0')
+packages = [
```

Comment:
is there a reason to have this fixed vs the dynamic find?

Reply:
It's automatically generated, so in a sense it is still dynamic :)